schema/deploy/shipping/views.sql

```diff
@@ -2954,7 +2954,7 @@ create or replace view shipping.uw_reopening_enrollment_fhir_encounter_details_v
   ,(select date_response[1] from shipping.fhir_questionnaire_responses_v1 where encounter_id = encounter.encounter_id and link_id = 'vaccine') as received_this_season_flu_vaccine_date
 
   from warehouse.encounter encounter
-  where encounter.details @> '{"_provenance": {"redcap": {"url": "https://redcap.iths.org/", "project_id":23854, "event_name":"enrollment_arm_1"}}}'
+  where encounter.details @> '{"_provenance": {"redcap": {"url": "https://hct.redcap.rit.uw.edu/", "project_id":45, "event_name":"enrollment_arm_1"}}}'
```

Comment:
I think all the enrollments are getting copied into the new project, so this should be ok to just pull from new. (But, if we aren't guaranteed to copy enrollment for all records, maybe we'd want to do something here to include both and then deduplicate on the invididual)